homeassistant/components/logbook.py

```diff
@@ -286,6 +287,32 @@ def humanify(hass, events):
                     'context_user_id': event.context.user_id
                 }
 
+            elif event.event_type == EVENT_HOMEKIT_CHANGED:
+                data = event.data
+                entity_id = data.get(ATTR_ENTITY_ID)
+                display_name = data.get('display_name')
+                message = data.get('message')
+
+                if message:
+                    action = "send command {}".format(message)
+                elif entity_id:
+                    action = "send command {}".format(entity_id)
+                else:
+                    action = "send command"
+
+                if display_name:
```

Comment:
The `display_name` will always be given, since the fallback is the `entity_id`. No need to check for it.

Reply:
Not completely following. Would you mind re-checking on what it looks like now (once I commit)?

Appreciate it!

Reply:
Will do.

What I was trying to say is that you will always have a `display_name`, so `if display_name` isn't necessary.